Front End/src/app/attendance/attendance.component.html

```diff
@@ -0,0 +1,49 @@
+<h1>Attendance</h1>
+
+<form [formGroup]="attendanceSeasonGameForm" (ngSubmit)="onSeasonGameSubmit()">  
+    <div class="form-inline">
+        <label class="control-label mr-2">Season</label>
+        <select class="form-control col-md-3 mr-2" formControlName="seasonNumber">
+            <option value="">Choose Season</option>
+            <option *ngFor="let season of allSeasons; let $index = index" [value]="allSeasons[$index]">{{season}}</option>
```

Comment:
Using `[value]="allSeasons[$index]"` over `[value]="season"` is superfluous, and the latter method cuts out the need to track the index